apps/wunda/verkehrszeichenkataster/src/components/commons/Navbar.jsx

```diff
@@ -26,10 +26,11 @@ import {
   storeAllApplications,
   storeSelectedApplications,
 } from "../../store/slices/application";
-import Logo from "/cismet.svg";
 import NavItem from "./NavItem";
 import "./input.css";
 
+const cismetLogoSrc = "/cismet.svg";
```

Comment:
@d4v3000 changed some logo and svg imports here please test. dont know where to look